README.md

```diff
@@ -222,8 +222,9 @@ const account = new Account(stack, "Account", {
 account.delegateAdministrator("stacksets.amazonaws.com");
 ```
 
-- To enable trusted access, you must have [all features](https://github.com/pepperize/cdk-organizations/blob/main/API.md#@pepperize/cdk-organizations.FeatureSet) enabled.
-- [AWS services that you can use with AWS Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_integrate_services_list.html)
+- [AWS services that support Delegated Administrator](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_integrate_services_list.html)
+- To be able to use Delegated Administrator, your organization must have all [all features](https://github.com/pepperize/cdk-organizations/blob/main/API.md#@pepperize/cdk-organizations.FeatureSet) enabled.
+- You must have enabled trusted access for Management Account in your organization.
```

Comment:
```suggestion
- To be able to use Delegated Administrator, your organization must have all [all features](https://github.com/pepperize/cdk-organizations/blob/main/API.md#@pepperize/cdk-organizations.FeatureSet) enabled.
```

Reply:
I'd haven't deleted the line You must have enabled trusted access for Management Account in your organization.https://docs.aws.amazon.com/accounts/latest/reference/using-orgs-delegated-admin.html

Reply:
Explain how to enable access for account management, i.e.:

> You must have enabled trusted access for Management Account in your organization: 
>  ```typescript
>  organization.enableAwsServiceAccess("account.amazonaws.com")
>  ```
>  See [Enable trusted access](https://github.com/pepperize/cdk-organizations#enable-an-aws-service-trusted-service)